renderer/canvas.py

```diff
@@ -231,11 +389,30 @@ def _draw_floor(self, surface):
         Args:
             surface (pygame.Surface): Surface to draw on
         """
-        floor_image_name = self.config["floor"]
-        clamped_pix_square_size = np.ceil(self.pix_square_size) # Necessary to avoid 1 pixel gaps from decimals
-        for row in range(len(self.layout)):
-            for col in range(len(self.layout[0])):
-                self._draw_image(surface, floor_image_name, np.array([col, row]) * clamped_pix_square_size, clamped_pix_square_size)
+        if not self.ground_tileset:
+            # load ground tile data
+            self.ground_tileset = self._load_tiles(self.config["floor"]["ground"])
+            self.ground_matrix = self._parse_abstract_tile_matrix(self.tiling["ground"], self.ground_tileset)
+
+        sprites = self.ground_tileset["sprites"]
+        
+        self._draw_tiles(surface, sprites, self.ground_matrix)
+
+    def _draw_furniture(self, surface):
```

Comment:
this functionality will need to be moved since the `furniture` is actually `stations` and the tilesets should be drawn based off the existing position of the stations